lua/entities/gmod_tardis/modules/sh_health.lua

```diff
@@ -272,7 +273,7 @@ if SERVER then
 			self:SetData("prevortex-flight", false)
 			self:Mat()
 		end
-		self:Explode()
+		self:Explode(300)
```

Comment:
What is the reason for increasing this from 60 to 300?

Reply:
This is the death scenario. Shouldn't death explosion be bigger than normal explosion due to technical difficulties?

Also, it would stop people from just shooting others' TARDISes up close. 
Intentionally or not, legacy had this feature that if you exploded the TARDIS up close, it backfired. This also works now here. It's not major, you can still shoot it from a distance.